src/cloudWatchLogs/commands/copyLogStreamName.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as vscode from 'vscode'
+import { parseCloudWatchLogsUri } from '../cloudWatchLogsUtils'
+
+export async function copyLogStreamName(uri?: vscode.Uri): Promise<void> {
+    if (!uri) {
+        uri = vscode.window.activeTextEditor?.document.uri
+        if (!uri) {
+            vscode.window.showErrorMessage(
+                localize('aws.cloudWatchLogs.invalidEditor', 'Current editor is not a valid Cloudwatch Logs editor.')
```

Comment:
same for this case. (Also: could avoid these tiny bits of duplicate code by defining a function-local function (i.e. not a module function)):

```
function doErr() {
            vscode.window.showErrorMessage(
                localize('aws.cloudWatchLogs.invalidEditor', 'Current editor is not a valid Cloudwatch Logs editor.')
}

...
doErr()
...
doErr()

```